src/components/Select.js

```diff
@@ -0,0 +1,30 @@
+export class Select {
+  #element;
+  #value;
+
+  constructor({ id, name, options, onChange, className }) {
+    this.#element = document.createElement("select");
+    this.#element.name = name;
+    this.#element.id = id;
+    this.#element.className = className;
+
+    this.#element.innerHTML = `${options
+      .map((option) => `<option value="${option}">${option}</option>`)
+      .join("\n")}`;
+
+    this.#value = options[0];
```

Comment:
이렇게 표현해줄 수 있도록 유틸함수를 하나 만들어주면 어떨까요?
```suggestion
    this.#element = createElement(`
      <select name="${name}" id="${id}" className="${className}">
        ${options
            .map((option) => `<option value="${option}">${option}</option>`)
            .join("\n")}
      </select>
    `)
```